bugbug/model.py

```diff
@@ -229,10 +229,11 @@ def get_top_features(cutoff, shap_values):
         return important_features
 
     def print_feature_importances(
-        self, important_features, feature_names, class_probabilities=None
+        self, important_features, class_probabilities=None
     ):
         # extract importance values from the top features for the predicted class
         # when classifying
+        feature_names = self.get_human_readable_feature_names()
```

Comment:
Please move this before the comment, as the comment is referring to the following code.